client/components/Routine.js

```diff
@@ -5,11 +5,10 @@ import {Card} from 'semantic-ui-react';
 const Routine = props => {
   const {routine, team} = props;
   let {id, title, url} = routine;
-  const teamId = props.match.params.teamId || routine.teamId;
+  const teamId = team.id;
```

Comment:
I think we had this just in case a user landed on the team page via a copy-pasted URL; can you triple check this would work if you copy-pasted the link to the routine?

Reply:
No worries, just checked and it should be alright. This would be on the the individual team page which should have a team loaded in on app mount